UKHomeOffice/sgmr-service
# SPCR-437 Add NotificationBanner to Dashboard

### AC / Description of changes
The dashboard no longer has a <NotificationBanner> which means that when you cancel a voyage the successfully cancelled voyage notification banner shows up on the PC page instead

### To Test
- Create a draft voyage
- Cancel the voyage
- > You should see a notification banner saying 'Voyage plan has been successfully cancelled.'

### Notes

---

**Remember to merge to feature branches not `master`.**
